app/views/shared/_side-nav.html.erb

```diff
@@ -47,6 +47,13 @@
           <% end %>
         </li>
       <% end %>
+
+      <!-- Draft Triage in progress -->
+      <% if session[:triage] %>
+      <li class="main-menu__item">
+        <%= link_to "Draft Triage (1)", contact_triage_path(session[:triage][:contact_id]), class: "main-menu__link main-menu__draft" %>
```

Comment:
What does the (1) mean here?  It's currently possible to add multiple triages for different people

Reply:
yes, maybe I should remove it

Reply:
done